google/data_source_google_folder_test.go

```diff
@@ -0,0 +1,79 @@
+package google
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceGoogleFolder(t *testing.T) {
+	skipIfEnvNotSet(t,
+		[]string{
+			"GOOGLE_ORG",
+		}...,
+	)
+
+	parent := fmt.Sprintf("organizations/%s", os.Getenv("GOOGLE_ORG"))
```

Comment:
you can use the global test variable `org` for this.

Reply:
Can you point me to the right place? Where does that live? I don't seem to be able to find it.